lib/streamlit/elements/utils.py

```diff
@@ -90,3 +90,19 @@ def get_label_visibility_proto_value(
         return LabelVisibilityMessage.LabelVisibilityOptions.HIDDEN
     elif label_visibility_string == "collapsed":
         return LabelVisibilityMessage.LabelVisibilityOptions.COLLAPSED
+
+
+def check_valid_button_type(button_type: str, widget_type: str) -> Any:
```

Comment:
I think this method can just be put into button.py(?). Is there a reason why you put it into utils? I don't think any other classes will use it?

Reply:
Personally, I found the button_type and widget_type parameters to be a little confusing. 

What about `button_type` and `button_class`? Maybe someone else has a better suggestion because when I read widget_type I expected other widgets could go into there? Maybe we could also just create a doc string that says what would go into widget_type?

Reply:
My thinking in putting it into utils was because it was shared between `button.py` and `form.py`, but I think you're right it probably makes more sense in `button.py` and I just import the function to `form.py`

Reply:
Yeah since this is really just limited to `st.button` and `st.form_submit_button` let me fix the param names & add a note for context